apps/payments/next/app/[offeringId]/checkout/page.tsx

```diff
@@ -0,0 +1,47 @@
+import { getContentfulContent, getCartData } from '../../lib/apiClient';
+import { PurchaseDetails, TermsAndPrivacy } from '@fxa/payments/ui/server';
+
+interface CheckoutParams {
+  offeringId: string;
+}
+
+export default async function Index({ params }: { params: CheckoutParams }) {
+  // TODO - Fetch Cart ID from cookie
+  // https://nextjs.org/docs/app/api-reference/functions/cookies
+  const cartId = 'cart-uuid';
+  // TODO - Fetch locale from params
+  // Possible solution could be as link below
+  // https://nextjs.org/docs/app/building-your-application/routing/internationalization
+  const locale = 'en-US';
+
+  const contentfulData = getContentfulContent(params.offeringId, locale);
+  const cartData = getCartData(cartId);
+  const [contentful, cart] = await Promise.all([contentfulData, cartData]);
+
+  return (
+    <>
+      <h1 className="page-title-container">Under Construction</h1>
+      <section className="payment-panel" aria-label="Purchase details">
+        <PurchaseDetails
+          interval={cart.interval}
+          invoice={cart.nextInvoice}
+          purchaseDetails={contentful.purchaseDetails}
+        />
+      </section>
+      <section
+        aria-label="Payment method and terms and privacy notices"
+        className="component-card border-t-0 min-h-full mb-6 pt-4 px-4 pb-14 rounded-t-lg text-grey-600 tablet:rounded-t-none desktop:px-12 desktop:pb-12"
+      >
+        <p className="h-[640px] flex items-center justify-center">
+          Section Under Construction
+        </p>
+        <TermsAndPrivacy
```

Comment:
I think it would be better to separate `TermsAndPrivacy` from the Payment method group.
cc: @sardesam, thoughts?

Reply:
Agreed. This is standalone content and should ideally not be grouped together with Payment method.